porch/engine/pkg/engine/testdata/context/expected/bucket.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: storage.cnrm.cloud.google.com/v1beta1
+kind: StorageBucket
+metadata:
+  name: bucket-name
+  namespace: bucket-namespace
+  annotations:
+    cnrm.cloud.google.com/project-id: bucket-project
+    config.kubernetes.io/path: 'bucket.yaml'
```

Comment:
Do we expect `path` annotations when a package is written out ?

Reply:
Yes for this output. The packages arrive from the server with annotations and are compared to golden files without modification for verbatim validation.